model/src/main/java/ru/training/karaf/model/BookDescriptionDo.java

```diff
@@ -0,0 +1,56 @@
+package ru.training.karaf.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
+public class BookDescriptionDo {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private String description;
+
+    private String date;
+
+    @OneToOne(mappedBy = "descriptionDo")
```

Comment:
I guess description without a book is not making much sense so let's make it not optional